java/README.md

```diff
@@ -2,20 +2,17 @@ Client requirement:
 java 1.8
 
 To build and run all tests:
-> ./gradlew build
+> ./gradlew build --offline
 
 To run the unit tests only:
-> ./gradlew test
+> ./gradlew test --offline
 
-To run the client, running all projections:
-> ./gradlew bootRun
+To run the application, create a fatJar first, then execute it
+> ./gradlew clean build fatJar --offline
+> java -jar build/libs/ddd-workshop-0.0.1-SNAPSHOT-all.jar
```

Comment:
Is it necessary to build the fatJar every time, or only after changing dependencies?